tower_cli/cli/resource.py

```diff
@@ -340,15 +339,13 @@ def get_command(self, ctx, name):
 
         # Make a click Command instance using this method
         # as the callback, and return it.
-        cmd = click.command(name=name, cls=ActionSubcommand, **attrs)(
-            new_method)
+        cmd = click.command(name=name, cls=ActionSubcommand, **attrs)(new_method)
 
         # If this method has a `pk` positional argument,
         # then add a click argument for it.
         code = six.get_function_code(method)
         if 'pk' in code.co_varnames:
-            click.argument('pk', nargs=1, required=False,
-                           type=str, metavar='[ID]')(cmd)
+            click.argument('pk', nargs=1, required=False, type=str, metavar='[ID]')(cmd)
```

Comment:
I see that you bought yourself some extra line length by moving this into a new file 😉